base/linalg/bidiag.jl

```diff
@@ -573,8 +573,8 @@ end
 factorize(A::Bidiagonal) = A
 
 # Eigensystems
-eigvals(M::Bidiagonal) = M.dv
-function eigvecs(M::Bidiagonal{T}) where T
+eigvals(M::Bidiagonal) = sort(M.dv, by=eigsortby)
+function eigvecs_(M::Bidiagonal{T}) where T # non-sorted eigenvectors
```

Comment:
yes, the fallback calls `eigfact(M)[:vectors]`, which gets the sorted eigenvectors.